src/bin/cli/ideascale/mod.rs

```diff
@@ -0,0 +1,138 @@
+use catalyst_toolbox::ideascale::{
+    build_challenges, build_fund, build_proposals, fetch_all, Error,
+};
+
+use structopt::StructOpt;
+
+use serde::Serialize;
+use std::fmt::{Display, Formatter};
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
+#[derive(Debug)]
+pub enum VoteType {
```

Comment:
This could reuse `vote::PayloadTypeDef` from jormungandr-lib.

Reply:
It is not exposed, iirc.

Reply:
It is not exposed actually